src/Protego.sol

```diff
@@ -103,4 +103,34 @@ contract Protego {
         DsPauseLike(pause).drop(_usr, _tag, _fax, _eta);
```

Comment:
Question @amusingaxl @oddaf: Should we reuse the  `Plan` struct and take a `Plan memory _plan` as this function's argument? Or do we keep it as it is?

Reply:
Let's keep it as it is.